sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/LiveMetricsExporter.cs

```diff
@@ -9,18 +10,28 @@ namespace Azure.Monitor.OpenTelemetry.LiveMetrics
     internal sealed class LiveMetricsExporter : BaseExporter<Metric>
     {
         private readonly string _instrumentationKey;
+        private readonly DoubleBuffer _doubleBuffer;
         private LiveMetricsResource? _resource;
         private bool _disposed;
 
-        public LiveMetricsExporter(LiveMetricsExporterOptions options)
+        public LiveMetricsExporter(DoubleBuffer doubleBuffer, LiveMetricsExporterOptions options)
         {
             _instrumentationKey = "";
+            _doubleBuffer = doubleBuffer;
         }
 
         internal LiveMetricsResource? MetricResource => _resource ??= ParentProvider?.GetResource().CreateAzureMonitorResource(_instrumentationKey);
 
         public override ExportResult Export(in Batch<Metric> batch)
         {
+            MonitoringDataPoint monitoringDataPoint = new MonitoringDataPoint();
```

Comment:
nit: this could be moved after line `28`

Reply:
This is just to show a sample, this part will change a lot. Leave it this way for now.